schemas/swagger/swagger.yaml

```diff
@@ -1438,7 +1438,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/CatapultUpgradeDTO'
+                $ref: '#/components/schemas/BlockchainUpgradeDTO'
```

Comment:
I remember approving a PR that has this change. Did we just merged that one with this one?

Reply:
I used latest swagger from rest api project. I have my own copy in repo